playfab-docs/features/authentication/aad-authentication/index.md

```diff
@@ -28,13 +28,13 @@ PlayFab now supports two methods of user authentication. The first is the origin
 
     ![Sign in with Microsoft](media/AADDoc3.png )
 
-### Sign-up with Azure AD
+### Sign-up with a Microsoft Authenticated Account
 
-You can start a new PlayFab studio via Azure AD.
+You can start a new PlayFab studio using a Microsoft account.
 
 1. Navigate to [developer.microsoft.com](https://developer.playfab.com/en-US/sign-up).
-2. Select **Sign in with Microsoft**.
+2. Select **Sign up with Microsoft**.
 
-### Azure AD Limitations
+### Microsoft Account Limitations
 
-Azure AD authentication is functional for individual users, including Azure AD token exchange for programmatic authentication. It doesn't support groups or graph.
+Microsoft Account authentication is functional for individual users, including Microsoft account token exchange for programmatic authentication. It doesn't support groups or graph.
```

Comment:
We seem to have inconsistent casing with Microsoft Account vs Microsoft account.

Reply:
Thanks. Fixed